README.md

```diff
@@ -80,6 +78,24 @@ import Select2 from 'react-select2-wrapper';
 />
 ```
 
+### Value
```

Comment:
Let this example will be after `Default multiple value`, please.
And text `Also possible to change the current value using value` so:

```
### Value

Also possible to change the current value using `value` property
```